MIP/mip-18/README.md

```diff
@@ -0,0 +1,99 @@
+# MIP-18: Stage 0 Upgradeability and Multisigs
+- **Description**: Proposal to establish two multisigs, one for engineers and another for the foundation, to facilitate the deployment and upgrade of the Movement Token (MOVEToken) and other upgradeable contracts.
+- **Authors**: [Primata](mailto:primata@movementlabs.xyz)
+- **Desiderata**:
+
+## Abstract
+
+MIP-18 advocates for the creation of two multisigs: one managed by Core Contributors engineers to propose contract upgrades and another managed by the Movement Foundation to execute them. This proposal enables testing of multisig services while ensuring secure and transparent upgrade processes for the Movement Token. The multisigs will be deployed on Ethereum, Sepolia, and Holesky (if available) networks.
+
+## Motivation
+
+There is an immediate need to deploy the Movement Token (MOVEToken) using a robust, secure, and auditable upgrade mechanism. By creating two multisigs—one for proposal and one for execution—we ensure a transparent and decentralized governance process over upgrades to critical contracts.
+
+The first multisig allows Core Contributors engineers to propose upgrades via Safe’s multisig service. A remote key will automate the population of upgrade proposals. The second multisig, held by the Movement Foundation, will be responsible for executing the scheduled upgrades after a time delay, ensuring community trust and review.
+
+Also, while developing an independent multisig, handling signature showed to be an issue that might be indicative of better letting signature to be handled by safe.global.
+
+## Specification
+
+### Multisig Setup
+
+1. **Core Contributors Multisig** (4/5 signatures required):  
+   - Role: Proposing upgrades.
+   - Addresses: [redacted], sep:0x493516F6dB02c9b7f649E650c5de244646022Aa0.
+   - Justification: We leave some wiggle room incase one of the engineers is away. We still need 3/4 engineers to sign.
+
+2. **Movement Foundation Multisig** (3/5 signatures required, triggerable after 2-day timelock):  
+   - Role: Executing upgrades.
+   - Addresses: [redacted], sep:0x00db70A9e12537495C359581b7b3Bc3a69379A00.
+
+3. **Deployer Multisig** (1/2 signatures required):
+   - Role: Deploy contracts.
+   - Addresses: [redacted].
+
+Both multisigs are part of a timelock mechanism to enforce a 2-day waiting period between the proposal and execution of upgrades. The engineers' multisig schedules the upgrade, while the foundation multisig executes it after the timelock expires.
```

Comment:
Once the timelock is set, can its value every be updated without a redeployment?

Reply:
I'm under the impression that maybe you could change it by using the fallback functionality, but maybe I'm wrong. That would require going through a transaction proposal to itself then executing it using the foundation multisig. I do believe that transferring to a new timelock is easier.